visual-challenge/src/main/java/applitools/hackathon/custom/DateCell.java

```diff
@@ -0,0 +1,16 @@
+package applitools.hackathon.custom;
+
+import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.interfaces.base.HasValue;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Text;
+
+import static java.lang.String.format;
+
+public class DateCell extends Section implements HasValue {
+    @UI("//span[1]") Text day;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4513258)